src_erl/Communication_Layer/http_Nerlserver/src/MainServer/initHandler.erl

```diff
@@ -20,13 +20,14 @@ init(Req0, [Main_genServer_Pid]) ->
 %%  can go to CSV file and edit weight
 
   %Bindings also can be accessed as once, giving a map of all bindings of Req0:
-  {ok,Body,_} = cowboy_req:read_body(Req0),
+  {_,Body,_} = cowboy_req:read_body(Req0, #{length => 10000000}),  %read up to 10MB (default was 8MB)
```

Comment:
Use -define to set it 
Read of up to 10MB of what? 
Better to set it in KB and multiply the code